CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ This change was introduced in [pull request #2694: Deprecate `.govuk-header__nav
 
 In [pull request 2678: Replace ex units with ems for input lengths](https://github.com/alphagov/govuk-frontend/pull/2678), we changed how we define input lengths in our CSS. Browsers might now display these inputs as being slightly wider than before. The difference is usually fewer than 3 pixels.
 
+In [pull request 2727: Make use of hidden in header navigation functionality](https://github.com/alphagov/govuk-frontend/pull/2727), we updated header component's mobile navigation functionality to better account for if CSS or javascript don't load. This feature doesn't require changes to your component code to continue working, however we recommend adding the attribute `hidden` to the mobile menu button (button element with class `govuk-header__menu-button`) to get the full benefits of the change.
```

Comment:
Made a best guess with this. It may not be the right place to put this and may not even need this level of detail.

Reply:
The 'recommended changes' from 4.1.0 might be good examples to pull from for this? 
https://github.com/alphagov/govuk-frontend/releases/tag/v4.1.0

Reply:
Updated. It'll still need a review fro ma content designer anyway and hinges a bit on the outcome of[ the discussion below](https://github.com/alphagov/govuk-frontend/pull/2727#discussion_r931077527) but we've got this ready to go if needs be.